package.json

```diff
@@ -345,6 +345,7 @@
     "webpack-cli": "3.3.12"
   },
   "dependencies": {
+    "@brave/leo": "github:brave/leo#928b9a6e381dfd2cb1728d590be168588040e73c",
```

Comment:
Looks like the Svelte PR upstream merged, so lets migrate back to their version and then this LGTM

Reply:
Decision was to keep using brave fork until upstream has pushed a version to npm which includes the PR